controllers/comment.js

```diff
@@ -1,16 +1,23 @@
-const { response } = require('express');
-const Comment = require('../models/Comment');
-const Todos = require('../models/Todo');
+const { response } = require("express");
+const Comment = require("../models/Comment");
+const Todo = require("../models/Todo");
+const Todos = require("../models/Todo");
```

Comment:
The import of the TOdos model is assigned twice in different variables, please remove one.